flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
what about this doesn't work on aarch64?

Reply:
I have no way to ensure that